js/about/preferences.js

```diff
@@ -591,6 +592,10 @@ class AboutPreferences extends React.Component {
       settings: this.state.settings.set(key, value)
     })
     aboutActions.changeSetting(key, value)
+    if (key === settings.DO_NOT_TRACK || key === settings.HARDWARE_ACCELERATION_ENABLED ||
+      key === settings.PDFJS_ENABLED) {
```

Comment:
@bbondy was there a reason you added the restart requirement for PDFJS in e962ef99cb6ad70ec743f8911abb7be304a4b1bc? it seems to work fine when i turn it on/off and don't restart